.github/workflows/release.yml

```diff
@@ -22,7 +22,8 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: ^1.16
+          go-version: '1.18'
```

Comment:
here as well: I think we want  go1.18, but not 100% sure

Reply:
lgtm